plugins/nf-azure/src/main/nextflow/cloud/azure/batch/AzBatchService.groovy

```diff
@@ -352,7 +353,9 @@ class AzBatchService implements Closeable {
 
         final slots = computeSlots(task, pool)
         log.trace "[AZURE BATCH] Submitting task: $taskId, cpus=${task.config.getCpus()}, mem=${task.config.getMemory()?:'-'}, slots: $slots"
-
+        List<EnvironmentSetting> environmentSettings = new ArrayList<>()
+        environmentSettings << new EnvironmentSetting().withName('DOCKER_CLIENT_TIMEOUT').withValue('120')
+        environmentSettings << new EnvironmentSetting().withName('COMPOSE_HTTP_TIMEOUT').withValue('120')
         final taskToAdd = new TaskAddParameter()
```

Comment:
Nice! worth adding a comment with a link to the discussion to highlight the need for those vars

Reply:
Done :)

Reply:
Awesome!